### DISCUSSION TITLE
Can I customize the slash menu and the overall appearance of novel?

### DISCUSSION THREAD

**Discussion by junoriosity** [steven-tey/novel#155] [Q&A]
@steven-tey @jaril @eyalcohen4 

When I embed the novel `Editor` as a React component, can I customize the appearance of novel overall?

For instance, it would be interesting to me to

- change/add new fields to the slash menu plus attached functionalities
- customize the stiling of some parts of the components
- customize the stiling of some "written" components like the boxes in "To do lists"

Is that possible and if so, how? Would be great to get some insights into it. 🙂

**Comment by sarimrmalik** [comment]
Would love this too @steven-tey
